invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/uploads_items/UploadsResultProps.js

```diff
@@ -0,0 +1,39 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2022 CERN.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import _get from "lodash/get";
+
+export const uploadsResultProps = (result) => {
```

Comment:
it's quite unorthodox in our code base to separate a function to provide the props
additionally, as a developer I wouldn't like to jump to another file to find out how my props are calculated.
most of these statements can be simplified, let me know if you need hints on how - please place them in the places they are used